src/js/controllers/ctrl-media-import.js

```diff
@@ -167,14 +186,17 @@ app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
 						creationDate.getFullYear(),
 					].join('/')
 
-					allowedResult.push({
-						id: res.id,
-						type: res.type,
-						name: fileName,
-						created: dateString,
-						timestamp: res.created_at,
-						thumb: _thumbnailUrl(res.id, res.type),
-					})
+					if (res.is_deleted == 0) {
```

Comment:
If this value specifically needs to be `0`, use `===`. If it just needs to be falsy, use `! res.is_deleted`.